src/lib/learn-client/schemas.ts

```diff
@@ -0,0 +1,136 @@
+import Joi from 'joi'
+
+import { ProductOption, ThemeOption } from './types'
+
+/**
+ * These schemas are used in testing to fully validate
+ * the shape returned from client functions
+ */
+
+// Page
+// (specifically product pages, which are the only type of page we have so far)
+
+const BrandedCalloutBlockSchema = Joi.object({
```

Comment:
Keeping the joi schemas for the blocks here as those will come in a subsequent PR.